### ISSUE TITLE
handleReset method also triggers handleAudioStop

### ISSUE THREAD

**Issue by gauravumrani** [sivaprakashDesingu/react-voice-recorder#24]
When handleReset method is triggered then it also calls handleAudioStop

**Comment by sivaprakashDesingu** [comment]
@gauravumrani 
Yeah, Once **Reset** triggered, Once Rest triggered we are calling the **handleAudioStop** to prevent the error of audio recorder start method. For example let say we are not calling the **handleAudioStop** call on reset action, if your try to start voice record again after reset the records will be keeps listen, when the user starts again we will get issue of **Aready Voice started**  so to prevent that issue we are calling the **handleAudioStop** method whenever the rest triggered.   
 
I hope I have given enought explanation. Let me if you have still doubt on this plz.
